src/main/java/network/brightspots/rcv/Tabulator.java

```diff
@@ -365,16 +365,26 @@ private void setWinningThreshold(Map<String, BigDecimal> currentRoundCandidateTo
               config.isHareQuotaEnabled()
                   ? config.getNumberOfWinners()
                   : config.getNumberOfWinners() + 1);
-      if (config.isNonIntegerWinningThresholdEnabled()) {
-        // threshold = (votes / (num_winners + 1)) + 10^(-1 * decimalPlacesForVoteArithmetic)
-        BigDecimal augend =
-            config.divide(
-                BigDecimal.ONE, BigDecimal.TEN.pow(config.getDecimalPlacesForVoteArithmetic()));
-        winningThreshold = config.divide(currentRoundTotalVotes, divisor).add(augend);
+      // If we use integers, we shouldn't use any decimal places.
+      // Otherwise we use the amount of decimal places specified by the user
+      int decimals =
+          config.isNonIntegerWinningThresholdEnabled()
+              ? config.getDecimalPlacesForVoteArithmetic()
+              : 0;
+      // Augend is the smallest unit comaptable our rounding.
```

Comment:
typos

Reply:
I see it should be "`compatible with our`." Is that all the typos you noticed?

Reply:
Yes, that's all!

Reply:
Got it.